src/test/suite/didactUriCompletionItemProvider.test.ts

```diff
@@ -276,7 +277,9 @@ async function executeCompletionTest(input: string, expected: string, selectLast
 	await vscode.workspace.fs.writeFile(testFileUri, Buffer.from(input));
 	const document = await vscode.workspace.openTextDocument(testFileUri);
 	const editor = await vscode.window.showTextDocument(document, vscode.ViewColumn.One, true);
-	await delay(500);
+	waitUntil( () => {
+		return editor != null;
```

Comment:
the editor value will always have the same value. it will never change between calls in the waitUntil.

What needs to be waited for?
That the editor has the focus? https://github.com/camel-tooling/vscode-camelk/blob/b02e7c2a2880b02c9b5baee02720f3c7dfc89003/src/test/suite/commands/NewIntegrationFileCommand.test.ts#L93
something else?

Reply:
Yeah, that might be a better check. We just want to make sure that the right text file is opened, since the test may have opened more than a single editor